### ISSUE TITLE
Unreadable input

### ISSUE THREAD

**Issue by Deraen** [SomMeri/less4j#267]
Hi,

Less4j is unable to read following the input which works in Less.js:

```less
span {
  padding-right: 0\9; // remove padding for < ie9 since default arrow can't be removed
}
```

I'm not sure if this is a documented feature for Less.js and I think this would place to use `~"0\9"` or `e("0\9")`. But an external less library is using this (https://github.com/thomaspark/bootswatch/blob/gh-pages/paper/bootswatch.less#L186) so I guess it would be nice to do the same thing as Less.js.

The exception:
```
❯ java -jar less4j-1.9.0.jar test.less 
Exception in thread "main" java.lang.NumberFormatException: For input string: "0\9"
	at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
	at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
	at java.lang.Double.parseDouble(Double.java:538)
	at java.lang.Double.valueOf(Double.java:502)
	at com.github.sommeri.less4j.core.parser.TermBuilder.setDoubleAndType(TermBuilder.java:181)
	at com.github.sommeri.less4j.core.parser.TermBuilder.buildFromNumber(TermBuilder.java:172)
	at com.github.sommeri.less4j.core.parser.TermBuilder.buildFromTerm(TermBuilder.java:86)
	at com.github.sommeri.less4j.core.parser.TermBuilder.buildFromChildTerm(TermBuilder.java:54)
	at com.github.sommeri.less4j.core.parser.TermBuilder.buildFromChildTerm(TermBuilder.java:49)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleTerm(ASTBuilderSwitch.java:144)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleTerm(ASTBuilderSwitch.java:99)
	at com.github.sommeri.less4j.core.parser.TokenTypeSwitch.switchOn(TokenTypeSwitch.java:88)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleExpression(ASTBuilderSwitch.java:153)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleExpression(ASTBuilderSwitch.java:99)
	at com.github.sommeri.less4j.core.parser.TokenTypeSwitch.switchOn(TokenTypeSwitch.java:81)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleExpression(ASTBuilderSwitch.java:153)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleExpression(ASTBuilderSwitch.java:99)
	at com.github.sommeri.less4j.core.parser.TokenTypeSwitch.switchOn(TokenTypeSwitch.java:81)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleExpression(ASTBuilderSwitch.java:153)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleExpression(ASTBuilderSwitch.java:99)
	at com.github.sommeri.less4j.core.parser.TokenTypeSwitch.switchOn(TokenTypeSwitch.java:81)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleDeclaration(ASTBuilderSwitch.java:303)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleDeclaration(ASTBuilderSwitch.java:99)
	at com.github.sommeri.less4j.core.parser.TokenTypeSwitch.switchOn(TokenTypeSwitch.java:73)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleBodyMembers(ASTBuilderSwitch.java:525)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.createGeneralBody(ASTBuilderSwitch.java:1112)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleGeneralBody(ASTBuilderSwitch.java:514)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleRuleSet(ASTBuilderSwitch.java:343)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleRuleSet(ASTBuilderSwitch.java:99)
	at com.github.sommeri.less4j.core.parser.TokenTypeSwitch.switchOn(TokenTypeSwitch.java:25)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleStyleSheet(ASTBuilderSwitch.java:137)
	at com.github.sommeri.less4j.core.parser.ASTBuilderSwitch.handleStyleSheet(ASTBuilderSwitch.java:99)
	at com.github.sommeri.less4j.core.parser.TokenTypeSwitch.switchOn(TokenTypeSwitch.java:53)
	at com.github.sommeri.less4j.core.parser.ASTBuilder.parse(ASTBuilder.java:26)
	at com.github.sommeri.less4j.core.ThreadUnsafeLessCompiler.doCompile(ThreadUnsafeLessCompiler.java:85)
	at com.github.sommeri.less4j.core.ThreadUnsafeLessCompiler.compile(ThreadUnsafeLessCompiler.java:76)
	at com.github.sommeri.less4j.core.ThreadUnsafeLessCompiler.compile(ThreadUnsafeLessCompiler.java:50)
	at com.github.sommeri.less4j.core.DefaultLessCompiler.compile(DefaultLessCompiler.java:50)
	at com.github.sommeri.less4j.commandline.CommandLine.compile(CommandLine.java:170)
	at com.github.sommeri.less4j.commandline.CommandLine.runAsSinglemode(CommandLine.java:87)
	at com.github.sommeri.less4j.commandline.CommandLine.realMain(CommandLine.java:68)
	at com.github.sommeri.less4j.commandline.CommandLine.main(CommandLine.java:32)

```

**Comment by SomMeri** [comment]
Thank you for reporting the bug. How critical is that for you?

**Comment by SomMeri** [comment]
Related less.js issue: https://github.com/less/less.js/issues/2207

**Comment by Deraen** [comment]
Hi,

Not very important I think. I'll try to get the 3rd party library to use CSS escape in their code.

**Comment by SomMeri** [comment]
Thank you. I plan to fix this, the `padding-right: 0\9` is valid css. It is just a question of whether it needs to be released right after or can wait until enough fixes collect.

**Comment by SomMeri** [comment]
Closed by commit 1b83c5df45a46010c585c1562dd34af6945cecec
